src/web_app_skeleton/Procfile.ecr

```diff
@@ -0,0 +1,2 @@
+web: bin/<%= project_name %>
```

Comment:
Let's add a spec line https://github.com/luckyframework/lucky_cli/blob/master/spec/integration/init_web_spec.cr#L19 that looks for `Procfile` to contain `bin/test-project`

Reply:
Good idea!